system/lib/compiler-rt/emscripten_exception_builtins.c

```diff
@@ -3,16 +3,17 @@
  * Emscripten is available under two separate licenses, the MIT license and the
  * University of Illinois/NCSA Open Source License.  Both these licenses can be
  * found in the LICENSE file.
- *
- * Support functions for emscripten setjmp/longjmp and exception handling
- * support.
- * See: https://llvm.org/doxygen/WebAssemblyLowerEmscriptenEHSjLj_8cpp.html
```

Comment:
Why remove  this comment?

Reply:
Thanks, I removed too much comment here, will restore.